Sources/Q42Stats/Q42Stats.swift

```diff
@@ -272,9 +270,7 @@ public class Q42Stats: NSObject {
 extension Q42Stats: WCSessionDelegate {
     @available(iOS 9.3, *)
     public func session(_ session: WCSession, activationDidCompleteWith activationState: WCSessionActivationState, error: Error?) {
-        DispatchQueue.main.async {
-            self._log(key: "Watch_paired", value: session.isPaired.description)
-        }
+        self._log(key: "Watch_paired", value: session.isPaired.description)
```

Comment:
How do we guarantee `self._log(` is only accessed from the same thread? 
1. Could we make `Q42Stats` an actor to isolate access to `var collected: [String: String]`? 
That'd interfere with the `WCSessionDelegate` conformance though…
2. Or wrap it in some `Box<T>`?
```swift
actor Box<T: Sendable> {
    private var value: T

    init(_ value: T) {
        self.value = value
    }

    func get() -> T {
        return value
    }

    func set(_ newValue: T) {
        value = newValue
    }
}
```
3. Or just mark the property `@MainActor` 🤷🏻‍♂️

Reply:
Marking it as `@MainActor` would have my preference since it's the simplest :)